src/main/java/stream/flarebot/flarebot/commands/moderation/PermissionsCommand.java

```diff
@@ -223,12 +226,13 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
 
                             String list = getStringList(groupList, page);
                             EmbedBuilder eb = MessageUtils.getEmbed(sender);
-                            eb.addField("Grou", list, false);
+                            eb.addField("Groups", list, false);
                             eb.addField("Current page", String.valueOf(page), true);
                             int pageSize = 20;
                             int pages =
                                     groups.size() < pageSize ? 1 : (groups.size() / pageSize) + (groups.size() % pageSize != 0 ? 1 : 0);
                             eb.addField("Pages", String.valueOf(pages), true);
+                            eb.addField("User", user.getAsMention(), false);
```

Comment:
I don't see the point in a field just for the user. Maybe make the embed smaller and use the title or something p

Reply:
I tried putting the user in the title and that didn't work out to well

Reply:
If you want to suggest something else go ahead

Reply:
If you want to suggest something else go ahead

Reply:
Username#discrim
Something like that, mentions doesn't work in a title.

Reply:
ok